_posts/2021-04-17-dd_00001_intro.md

```diff
@@ -0,0 +1,324 @@
+---
+layout: single
+title: Introduction to Digital Design
+category: Digital Design
+permalink: /categories/digital_design/intro
+excerpt: "A gentle introduction to the field of digital chip design"
+author: Kirthihan Yasotharan
+toc: true
+toc_label: "Content"
+toc_sticky: true
+toc_icon: "align-justify"
+#sidebar:
+#  nav: "getting_started_0"
+header:
+  overlay_color: "#000"
+  overlay_filter: "0.5"
+  overlay_image: /assets/images/digital_design.jpg
+---
+
+Welcome to our first post on **Digital Design**! We are glad that you found your
+way into this very exciting topic. In this post, we will try to give you an
+overview of the various aspects of digital design. For each topic, we will of
+course make a detailed and comprehensive post. This series intends to share our
+knowledge and experience in designing digital systems with you. Unlike many
```

Comment:
digital systems such as ...

I think someone who never heard about digital design does not know what you mean by digital systems. So, just give one or more examples

Reply:
f1bf041
Like this?